sdks/java/testing/load-tests/src/main/java/org/apache/beam/sdk/loadtests/LoadTestOptions.java

```diff
@@ -37,6 +38,22 @@
 
   void setStepOptions(String stepOptions);
 
+  @Description("Whether the results should be published to BigQuery database")
+  @Default.Boolean(false)
+  Boolean getPublishToBigQuery();
+
+  void setPublishToBigQuery(Boolean publishToBigQuery);
+
+  @Description("BigQuery dataset name")
+  String getBigQueryDataset();
+
+  void setBigQueryDataset(String dataset);
+
+  @Description("BigQuery table name")
+  String getBigQueryTable();
+
+  void setBigQueryTable(String tableName);
+
```

Comment:
What will happen when the options conflict? If there's an exception thrown then we are good now I guess. If not, it may be wise to change it for Nexmark too, WDYT?

Reply:
I am not sure what happens when the options conflict... But I am okay with leaving this for later.